README.md

```diff
@@ -44,6 +44,8 @@ of our latest development branch.
 
 Note: RMM is supported only on Linux, and with Python versions 3.6 or 3.7.
 
+Note: If you choose to install RMM using Conda, make sure to use GCC 7 or later. Otherwise, your application may fail to build.
```

Comment:
Instead of this, let's just change the GCC compiler minimum requirement below to 7.0 or higher (and drop "recommended").  I was not aware that RMM does not build with GCC 5.x anymore because we do not test it (in CI) below 7.x anymore AFAIK.